apps/judicial-system/web/src/routes/CourtOfAppeal/Overview/Overview.tsx

```diff
@@ -28,6 +28,7 @@ import useAppealAlertBanner from '@island.is/judicial-system-web/src/utils/hooks
 import * as constants from '@island.is/judicial-system/consts'
 
 import { courtOfAppealOverview as strings } from './Overview.strings'
+import { UserRole } from '@island.is/judicial-system-web/src/graphql/schema'
```

Comment:
Consider ordering import so that "full path" imports come before "relative" imports.